engine/src/CalciteInterpreter.cpp

```diff
@@ -217,22 +212,22 @@ project_plan_params parse_project_plan(blazing_frame & input, std::string query_
 
 			output_columns.push_back(output.get_gdf_column());
 
-			add_expression_to_plan(input,
-				input_columns,
-				expression,
-				cur_expression_out,
-				num_expressions_out,
-				input_columns.size(),
-				left_inputs,
-				right_inputs,
-				outputs,
-				operators,
-				unary_operators,
-				left_scalars,
-				right_scalars,
-				new_column_indices,
-				final_output_positions,
-				output.get_gdf_column());
+			// add_expression_to_plan(input,
```

Comment:
is this supposed to be commented? whre does this happen now?

Reply:
this is being tackle in #247